packages/types/src/telemetry.ts

```diff
@@ -0,0 +1,42 @@
+import type { InstanceType } from 'instance';
+
+export type TelemetryEvent = {
+  event: string;
+  /**
+   * publishableKey
+   */
+  pk?: string;
+  /**
+   * secretKey
+   */
+  sk?: string;
+  /**
+   * instanceType
+   */
+  it: InstanceType;
+  /**
+   * clerkVersion
+   */
+  cv: string;
+  /**
+   * SDK
+   */
+  sdk?: string;
+  /**
+   * SDK Version
+   */
+  sdkv?: string;
+  payload: Record<string, string | number | boolean>;
+};
+
+export type TelemetryEventRaw<Payload = TelemetryEvent['payload']> = {
+  event: TelemetryEvent['event'];
+  eventSamplingRate?: number;
+  payload: Payload;
+};
```

Comment:
Now that we are exposing these as public API i feel that we need to do the following

Rename `TelemetryEventRaw` to `TelemetryEvent`
Rename `TelemetryEvent` to `TelemetryRecordedEvent` or `TelemetryStoredEvent`

Reply:
I agree that `Raw` seems a bit generic here 🤔 - @BRKalow do you have other suggestions on that naming?

Reply:
🤔 I wouldn't _really_ consider these part of the public API, it's an internal type that we're forced to expose due to how our packages are structured...

Raw feels accurate because it's the "raw" event that we're collecting before transforming it to send to our backend. `TelemetryRecordedEvent` is fine if we want to rename that.

We could also put a JSDoc with `@internal` to make it clear these aren't meant for public usage:
```ts
/**
 * @internal
 */
export type TelemetryEventRaw = ...
```

Reply:
> Raw feels accurate because it's the "raw" event that we're collecting before transforming it to send to our backend. TelemetryRecordedEvent is fine if we want to rename that.

@BRKalow ACK - thanks for the context!  I'm going to move forward with adding `@internal` to those interfaces.